
# 0: left 1: right
int16 camera_info
bool camera_check
